ote/src/cljs/ote/app/controller/transit_visualization.cljs

```diff
@@ -114,7 +120,9 @@
                                     route-changes
                                     group-recent?)
         route-changes (map (fn [x]
-                             (assoc x :count (count-changes (:route-hash-id x) grouped-changes)))
+                             (-> x
+                                 (assoc :count (count-changes (:route-hash-id x) grouped-changes))
+                                 (assoc :combined-change-types (combine-change-types (get grouped-changes (:route-hash-id x))))))
```

Comment:
Does :combined-change-types need to be a string? 
Should we prefer operations using keywords instead of strings?
Would a `set` or `list` suit as well? Then `contains?` etc. queries could use keywords without string operations.
If :change-type is a string in root map, it could be converted here via `:keyword` before conjoining to the collection.